app/config.py

```diff
@@ -0,0 +1,17 @@
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+    database_hostname: str
+    database_password: str
+    database_username: str
+    database_name: str
+    database_port: str
+    # secret_key: str
+    # algorithm: str
+    # access_token_expire_minutes: int
+
+    class Config:
+        env_file = ".env.py"
```

Comment:
[python-dotenv](https://pypi.org/project/python-dotenv/) library may be useful here